ReactNativeClient/lib/components/global-style.js

```diff
@@ -153,6 +153,8 @@ function themeStyle(theme) {
 
 		output.colorUrl = '#7B81FF';
 
+		output.strongTextColor = 'rgb(220,220,220)';
```

Comment:
This should be `output.colorBright`. Unless I'm missing some reason for adding a new theme variable.

Reply:
Sir, I changed the variable name to color bright. I added a new variable because I thought color bright won't be a good name for all themes, as it sounds like it's for the light theme. 
Please check it now.
Thanks